dashboards/Threat Intel.json

```diff
@@ -194,18 +188,18 @@
                                         "aggregates": [
                                             {
                                                 "operator": "select",
-                                                "alias": "typeOfFeed",
-                                                "field": "typeOfFeed.itemValue"
+                                                "alias": "type",
+                                                "field": "type.itemValue"
                                             },
                                             {
                                                 "operator": "select",
                                                 "alias": "color",
-                                                "field": "typeOfFeed.color"
+                                                "field": "type.color"
```

Comment:
this will still be typeOfFeed

Reply:
this dashboard no longer needed. so removed it.